content/includes/survey-tools.html

```diff
@@ -1,4 +1,4 @@
-{% if buildtype == 'staging' %}
+{% if buildtype == 'production' %}
```

Comment:
Is this only going to be included on prod now, and now longer be on staging?

Reply:
Yeah I think that's the intent.

Reply:
Cool. Just wanted to make sure that was intentional!